test/code-fixtures/projects/app/tsconfig.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
I consolidated all of the tsconfig.json contents into the one under "projects". This simplified the test cases that scanned from that directory level and didn't lose any test coverage.